tests/chainer_tests/functions_tests/evaluation_tests/test_accuracy.py

```diff
@@ -12,6 +13,34 @@
 from chainer.utils import type_check
 
 
+def _testing_pairwise(*parameters):
+    rs = numpy.random.RandomState(seed=0)
+    parameters = [list(enumerate(dicts)) for dicts in parameters]
+
+    uncovered = {}
+    for (i, dicts_i), (j, dicts_j) in itertools.combinations(
+            enumerate(parameters), 2):
+        uncovered[(i, j)] = set(itertools.product(
+            range(len(dicts_i)), range(len(dicts_j))))
+
+    product_dicts = list(itertools.product(*parameters))
+    rs.shuffle(product_dicts)
```

Comment:
Why is this shuffle required?

Reply:
The shuffle reduces the number of tests.  The following improvements may address your concern.
- sort selected tests, and/or
- output indices as they were generated by `testing.parameterize`